recipes/py-graphviz/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.9" %}
+{% set sha256 = "865afa6ab9775cf29db03abd8e571a164042c726c35a1b3c1e2b8c4c645e2993" %}
+
+package:
+  name: py-graphviz
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.python.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: $PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .
```

Comment:
mh, just try `python` here

Reply:
@bgruening thanks for the review.

I assume `python` `$PYTHON` and `{{ PYTHON }}` differ only in when the PATH is evaluated and using `python` is a matter of preference.

https://github.com/conda-forge/staged-recipes/blob/f46dacf1e9451f6c97d4f603a1bfaa0b78cadf99/recipes/example/meta.yaml#L33